backend/app/services/implementations/onboarding_request_service.py

```diff
@@ -0,0 +1,42 @@
+from ..interfaces.onboarding_request_service import IOnboardingRequestService
+from ...models.onboarding_request import OnboardingRequest
+from ...models.user_info import UserInfo
+
+# from ...resources.onboarding_request_dto import OnboardingRequestDTO  # TODO
```

Comment:
Nit: is this still a TODO? Looks like this file exists

Reply:
I commented it out because we weren't sure if we needed the DTO. Will remove it for now.